### DISCUSSION TITLE
关于mysql使用ado的问题

### DISCUSSION THREAD

**Discussion by zybwzy** [dotnetcore/FreeSql#1468] [General]
string sql = "SELECT su.* FROM " +
            "(SystemUser su inner join UsersRelation ur on su.Id = ur.UserID " +
            "inner join Organize o2 on ur.OrganizeID = o2.Id) where o2.Id = @id";
        var systemUsers = _freeSql.Ado.Query<SystemUser>(sql,id);    -->这里给id赋值不好使

**Comment by luoyunchong** [comment]
不同驱动的参数化前缀不同

https://freesql.net/guide/ado.html#%E5%8F%82%E6%95%B0%E5%8C%96

而且参数化是

, new { id = 1 }

**Comment by zybwzy** [comment]
感谢


| |
zhaoyabo0518
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年3月22日 15:07 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [dotnetcore/FreeSql] 关于mysql使用ado的问题 (Discussion #1468) |

https://freesql.net/guide/ado.html#%E5%8F%82%E6%95%B0%E5%8C%96
